cmd/storj-sim/prefix.go

```diff
@@ -84,7 +84,7 @@ func (writer *prefixWriter) Write(data []byte) (int, error) {
 
 	// buffer everything that hasn't been written yet
 	if len(writer.buffer) > 0 {
-		buffer = append(writer.buffer, data...)
+		buffer = append(writer.buffer, data...) //nolint
```

Comment:
I would exclude only `gocritic` not all the linters

reference: https://github.com/golangci/golangci-lint#false-positives

Reply:
Good catch.  resolved an pushed